Fix pytube#1154

RonyMacfly opened the pull request.

RonyMacfly committed:
Fix pytube#1154

yerycs commented:
It is not working correctly for auto generated caption

RonyMacfly commented:
@yerycs I'm sorry I haven't responded for a long time. 
Can you give me a link to the video where the error occurs?

yerycs commented:
https://www.youtube.com/watch?v=9WRML0bwoyo
You can check auto generated captions from above url

RonyMacfly committed:
get captions as an array

from pytube import YouTube
yt = YouTube("https://www.youtube.com/watch?v=AcPqRJl-jCk")
#video = yt.captions["en"]
video = yt.captions["a.en"]
print(video.get_captions_as_array())
